### DISCUSSION TITLE
OpenID with Authentik (Something went wrong while processing your request.)

### DISCUSSION THREAD

**Discussion by Mon-sceau** [seriousm4x/UpSnap#643] [Help wanted] [answered]
Hi,

I've configured Authentik for use it with UpSnap.
When I click on "authentik" it redirects me to authentik, then it tells me "authentication successful" and I'm redirected to the upsnap page with this error message : _"Something went wrong while processing your request."_

I think the problem comes from the "User API URL" in the configuration but i don't understand what i should put here. 
I already try this : _https://upsnap.mydomain.com/api/collections/users/auth-with-oauth2_ and _https://upsnap.mydomain.com/api/oauth2-redirect_

Does this configuration require any additional setups?

Some help? 
Thanks in advance

**Comment by seriousm4x** [comment]
I have not used OIDC/Authentik yet. I think you are better off asking Pocketbase directly in the [discussions](https://github.com/pocketbase/pocketbase/discussions/categories/q-a).

Based on [this image](https://github.com/pocketbase/pocketbase/discussions/3799#discussioncomment-7828549) I think you need to put some authentik user api url in there. Not an upsnap url.

**Comment by Mon-sceau** [comment]
Thanks, i will try that

**Comment by Mon-sceau** [comment, marked as answer]
Ok finnaly i found, it's simply the UserInfo URL : _**https://authentik.example.com/application/o/userinfo/**_
_**https://upsnap.mydomain.com/api/oauth2-redirect**_ is the Callback URL
